dist/types/accountBillingStatus.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+/**
+ * Billing status for a Tozny account
+ */
+class AccountBillingStatus {
+    constructor(accountActive, isTrial, trialPeriodEnds, isLegacy, isGoodStanding = null) {
+        this.accountActive = accountActive;
+        this.isTrial = isTrial;
+        this.trialPeriodEnds = trialPeriodEnds;
+        this.isLegacy = isLegacy;
+        this.isGoodStanding = isGoodStanding;
+    }
+    /**
+     * Specify how an already unserialized JSON array should be marshaled into
```

Comment:
Is this needed?

Reply:
this and other things happening in `dist` are the compiled code. i intentionally limited any changes to existing source code. i found the `decode` to class pattern at use in `types` strange. i think it's because other parts of the code make use of `instanceof` checks that require the js object to be marshaled into a class instance.

i think we could eventually move away from this pattern and rely on typesafety of the objects themselves in the future.